subscriptions.go

```diff
@@ -132,6 +132,7 @@ type SubscriptionAddOn struct {
 type PendingSubscription struct {
 	XMLName            xml.Name            `xml:"pending_subscription"`
 	Plan               NestedPlan          `xml:"plan,omitempty"`
+	UnitAmountInCents  int                 `xml:"unit_amount_in_cents"`
```

Comment:
Sorry, I missed this. Could you add `omitempty` here? Sometimes there are issues with empty items in structs when they are unmarshaled in XML.